io.github.sil_car.SqueezeVid.yaml

```diff
@@ -0,0 +1,58 @@
+# https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest
+app-id: io.github.sil-car.SqueezeVid
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: squeeze-vid
+
+finish-args:
+  # - --env=PYTHONPATH=$PYTHONPATH:/app/lib/python3.10/site-packages/squeeze_vid
+  - --filesystem=home
+
+modules:
+  - name: libsvtav1
+    # Ref: https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Build-Guide.md#1-build-and-install-svt-av1
+    sources:
+      - type: git
+        url: https://gitlab.com/AOMediaCodec/SVT-AV1.git
+        tag: v1.4.1
+    # source-depth: 1 "shallow clone" is used by default
+    buildsystem: cmake
```

Comment:
I"ll assume that it doesn't work with Ninja.

Reply:
I can look into that. Most of the modules' details came from me getting this built as a snap package. I'm still figuring out the differences between snapcraft and flatpak-builder.

Reply:
These buildsystem parameters came from SVT-AV1's [Build Guide](https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Build-Guide.md#1-build-and-install-svt-av1).

Reply:
```suggestion
    buildsystem: cmake-ninja
```